projects/1.16/assets/dimensional-dungeons/dimdungeons/lang/zh_cn.json

```
@@ -1 +1,289 @@
-{}
\ No newline at end of file
+{
+	"itemGroup.dimdungeons": "维度地牢",
+	"biome.dimdungeons.biome_dungeon": "地牢",
+	"dimension.dimdungeons.dimension_dungeon": "地牢维度",
+
+	"block.dimdungeons.block_gilded_portal": "镀金传送门方块",
+	"block.dimdungeons.block_gold_portal": "黄金地牢传送门",
+	"block.dimdungeons.block_local_teleporter": "入口传送门",
+	"block.dimdungeons.block_portal_keyhole": "传送门基石",
+	"block.dimdungeons.block_portal_crown": "维度皇冠座",
+	"block.dimdungeons.block_key_charger": "钥匙刻字台",
+	"block.dimdungeons.block_key_charger_used": "已使用的钥匙刻字台",
+	"block.dimdungeons.block_key_charger_damaged": "损坏的钥匙刻字台",
+
+	"item.dimdungeons.item_portal_key": "空白传送门钥匙",
+	"item.dimdungeons.item_portal_key_basic": "基础维度密钥",
+	"item.dimdungeons.item_portal_key_advanced": "高级维度密钥",
+	"item.dimdungeons.item_secret_bell": "秘密之钟",
+	"item.dimdungeons.item_homeward_pearl": "归途明珠",
+	"item.dimdungeons.item_guidebook": "维度地牢指南",
+	
+	"item.dimdungeons.item_trophy_1": "1UP蘑菇",
+	"item.dimdungeons.item_trophy_2": "大师剑",
+	"item.dimdungeons.item_trophy_3": "伊斯特符文",
+	"item.dimdungeons.item_trophy_4": "泰拉瑞亚",
```

```suggestion
	"item.dimdungeons.item_trophy_4": "泰拉悠悠球",
```
参考自[泰拉瑞亚wiki](https://terraria.fandom.com/zh/wiki/%E6%B3%B0%E6%8B%89%E6%82%A0%E6%82%A0%E7%90%83)，具体应视游戏内内容而定